docs/content/operations/platforms/kubernetes/kubernetes-metadata/snippets/env.bicep

```
@@ -14,34 +13,59 @@ resource env 'Applications.Core/environments@2022-03-15-privatepreview' = {
       {
         kind: 'kubernetesMetadata'
         labels: {
-          'myapp.team.name':'Operations'
-          'myapp.team.contact':'support-operations'
-        }
-        annotations: {
-          'prometheus.io/scrape': 'true'
-          'prometheus.io/port': '9090'
+          'key1': 'envValue1'
+          'key2': 'envValue2'
         }
       }
     ]
   }
 }
+//ENV
 
+//APP
 resource app 'Applications.Core/applications@2022-03-15-privatepreview' = {
-  name: 'webapp'
+  name: 'myapp'
   location: 'global'
   properties: {
-    environment: environment
-}
+    environment: env.id
+    extensions: [
+      {
+        kind: 'kubernetesMetadata'
+        labels: {
+          'key1': 'appValue1'
+          'key2': 'appValue2'
+          'team.contact.name': 'Operations'
+          'team.contact.email': 'ops@example.com'
```

```suggestion
          'team.contact.email': 'ops'
```
'@' is not supported by Kubernetes labels regex